octokit/meta.go

```diff
@@ -0,0 +1,41 @@
+package octokit
+
+import (
+	"net/url"
+
+	"github.com/jingweno/go-sawyer/hypermedia"
+)
+
+// https://developer.github.com/v3/meta/
+var (
+	MetaURL = Hyperlink("/meta")
+)
+
+// https://developer.github.com/v3/meta/
+func (c *Client) Meta(url *url.URL) *MetaService {
+	return &MetaService{client: c, URL: url}
+}
+
+// A service to return information about GitHub.com, the service.
+type MetaService struct {
+	client *Client
+	URL    *url.URL
+}
+
+// Get the user search results based on MetaService#URL
+//
+// https://developer.github.com/v3/meta/#meta
+func (m *MetaService) One() (meta Meta, result *Result) {
+	result = m.client.get(m.URL, &meta)
+	return
+}
+
+type Meta struct {
```

Comment:
I think it depends on what we do for the method name. https://github.com/octokit/go-octokit/pull/118/files#r36671782